test/helpers/configuration.ts

```diff
@@ -0,0 +1,21 @@
+import request, { Test } from 'supertest';
+import _ from 'lodash';
+import { hookRequest } from './hooks';
+
+/**
+ * Makes an admin request to configure the log level for Harmony
+ *
+ * @param app - The express application (typically this.frontend)
+ * @param options - Mapping object. Includes (optional) query param object (which maps query
+ * param names to values) and (optional) username.
+ * e.g. \{username: 'billy', query: \{...\}\}
+ */
+export function configureLogLevel(
+  app: Express.Application,
+  options: { username?: string; query?: object },
+): Test {
+  const { query } = options;
+  return request(app).get('/admin/configuration/log-level').query(query || {});
```

Comment:
I think the more idiomatic way is to set the default in the parameters - ie:

```
options: { username?: string; query?: object = {} },
```

Reply:
There was some error about an object literal not being allowed to have an initializer but I got rid of the `|| {}` as it's not really necessary anyway.